### DISCUSSION TITLE
Does suspend not work?

### DISCUSSION THREAD

**Discussion by txtyash** [nushell/nushell#5239] [Q&A]
Every time I try using `ctrl+z`, something weird happens. I can't even describe. Nushell is sometimes unable do suspend some cli apps and sometimes it just shows me the command prompt but just accepts text input but doesn't run.

**Comment by kubouch** [comment]
We don't support background tasks yet but it's something we want to look at.

**Reply by jeanlucthumm** [reply]
Is there a tracking issue for this?

**Reply by weirdan** [reply]
See https://github.com/nushell/nushell/issues/247
